src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -588,6 +588,21 @@ default Color sarcophagusOtherPurpleRecolor()
 		return new Color(17, 88, 152);
 	}
 
+	String CURSED_PHALANX_DETECT = "cursedPhalanxDetect";
```

Comment:
Since the config key is not referenced dynamically (via ConfigManager methods), just inline the value below instead of having a named constant.